Assets/Scripts/Tank/TankService.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TankService : GenericSingleton<TankService>
+{
+    public TankView tankView;
+
+    protected override void Awake()
+    {
+        base.Awake();
+    }
+
+    private void Start()
+    {
+        StartGame();
+    }
+
+    private void StartGame()
+    {
+        for(int i = 0; i < 3; i++)
+        {
+            CreateNewTank();
+        }
+    }
+
+    private TankController CreateNewTank()
+    {
+        TankModel tankModel = new TankModel(10, 100f);
```

Comment:
Give these values in constant  one

Reply:
private tankValue = 10f;
 TankModel tankModel = new TankModel(tankValue, 100f);